src/components/Modal/BottomDockedModal/Container/index.web.tsx

```diff
@@ -8,18 +8,28 @@ const easing = Easing.bezier(0.76, 0.0, 0.24, 1.0).factory();
 
 function Container({style, animationInTiming = 300, animationOutTiming = 300, onOpenCallBack, onCloseCallBack, ...props}: ModalProps & ContainerProps) {
     const styles = useThemeStyles();
+    const opacity = useSharedValue(0);
+    const isInitiated = useSharedValue(false);
 
-    const Entering = useMemo(() => {
-        const FadeIn = new Keyframe({
-            from: {opacity: 0},
-            to: {
-                opacity: 1,
-                easing,
```

Comment:
We used keyframes here for a reason - so we could use this easing defined above, after this change the easing isn't used anymore. I'm guessing the entering animation might look slightly different now